src/faucet-frontend.js

```diff
@@ -0,0 +1,50 @@
+const {Conflux} = require('js-conflux-sdk');
+const BigNumber = require('bignumber.js');
+const fs = require('fs');
+const faucet = JSON.parse(
+    fs.readFileSync(__dirname + '/../build/contracts/SponsorFaucet.json'),
+);
+const price = 111;
```

Comment:
better to allow client to specify gas price.

Reply:
fixed